charts/enterprise-kyverno-operator/templates/imagepullsecret.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: v1
 kind: Secret
 metadata:
   labels: {{ include "enterprise-kyverno.labels" $ | nindent 4 }}
```

Comment:
is this correct? `enterprise-kyverno.labels`

Reply:
Yes, this is fine. I have not changed variables, since those were a lot, and was not confident of a bulk search replace.